### ISSUE TITLE
 New Steam Client causes hard reset of system ingame 

### ISSUE THREAD

**Issue by DASPRiD** [ValveSoftware/steam-for-linux#9653]
#### Your system information

* Steam client version (build number or date): 1686880776
* Distribution (e.g. Ubuntu):  PopOS 22.04
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* GPU: Nvidia RTX 2070

#### Please describe your issue in as much detail as possible:
Since the new Steam client update on June 14 I am experiencing constant hard reset issues while being in VRChat, both in Desktop mode and VR. It usually happens when loading into specific worlds or loading specific avatars, but today I experienced it by just idling in an empty world after a while. The hard reset is mostly followed by a short complete system freeze.

I cannot see anything related to this in the syslog or any other log. My assumption would be that the libraries distributed with the new Steam client trigger some bug in the graphics driver, which forced the system to hard reset, though I do not have any concrete evidence for that. All I know is that the Steam client update was the only update of relevant applications/libraries since this issue started to occur.

My usual way of verifying this would be to roll the Steam installation back to pre June 14, but I haven't found a way yet to download an older version.

As for system information, I' running an RTX 2070 on driver version 525.116.04.

**Comment by kisak-valve** [comment]
Hello @DASPRiD, in general, no OpenGL application including Steam should be capable of taking down the system. This reads more like a video driver issue than a Steam client issue. We know the most recent Steam client update is asking more from the video driver in terms of hardware acceleration.

This issue should also be mentioned to your video driver vendor. I would check dmesg and the system logs for any hints around the time of the incident.

**Comment by DASPRiD** [comment]
I totally agree that graphics software should not be able to cause that at all. As I said, there is absolutely nothing in the syslog when it comes to the reset. I just installed the latest 535 driver via the graphics driver PPA and will test if this affects the issue.

**Comment by DASPRiD** [comment]
I just let VRChat run for a while in the background with the latest NVIDIA 535 driver and still experienced the crash. Interestingly not while idling, but once I started a YouTube video in a browser after about 5 seconds. Interestingly, I had a crash like that before on the 525 driver by just having Steam itself running but no game.

First the Xorg server freezes for about 5-10 seconds (though mouse cursor is still responsive), then eventually the PC powercycles by itself.

I re-checked the syslog after the crash, but there is absolutely nothing in there.

Is there any way to obtain the pre-june Steam client for testing whether it's actually that?

**Comment by DASPRiD** [comment]
I just let VRChat run for a while in the background with the latest NVIDIA 535 driver and still experienced the crash. Interestingly not while idling, but once I started a YouTube video in a browser after about 5 seconds. Interestingly, I had a crash like that before on the 525 driver by just having Steam itself running but no game.

First the Xorg server freezes for about 5-10 seconds (though mouse cursor is still responsive), then eventually the PC powercycles by itself.

I re-checked the syslog after the crash, but there is absolutely nothing in there.

-- Edit --
After a restart I experienced the same freeze + power cycle without steam even running, just by watching a YouTube video. I guess that excludes Steam as the culprit.

**Comment by gnollhide** [comment]
I'm not certain, I've had this happen literally twice, back-to-back, when attempting to join a game in Deep Rock Galactic. Both times happened after the new client was pushed out, and does NOT seem to be happening for any linux-native games. Exact same situation, the entire desktop freezes for 5-10 seconds (with associated audio being a short, broken loop of the last thing playing), then a hard crash to BIOS and restart.

**Comment by DASPRiD** [comment]
@gnollhide Okay, a second user being affected by the same issue is kinda strange. If it was just me, I'd put it on a very unlucky timing for a hardware defect, but two…

What's your system setup / OS? I wonder if we share some similarities.

**Comment by gnollhide** [comment]
Ubuntu OS, using NVIDIA drivers and apt-based steam install, let me go get you the output of neofetch on there and I'll post it in reply.

**Comment by gnollhide** [comment]
neofetch
`gnoll@ubuntubox:~$ neofetch
           `:+shmNNMMNNmhs+:`              gnoll@ubuntubox 
        .odMMMMMMMMMMMMMMMMMMdo.           --------------- 
      /dMMMMMMMMMMMMMMMmMMMMMMMMd/         OS: Ubuntu MATE 22.04.2 LTS x86_64 
    :mMMMMMMMMMMMMNNNNM/`/yNMMMMMMm:       Host: MS-7793 2.0 
  `yMMMMMMMMMms:..-::oM:    -omMMMMMy`     Kernel: 5.15.0-75-generic 
 `dMMMMMMMMy-.odNMMMMMM:    -odMMMMMMd`    Uptime: 2 hours, 42 mins 
 hMMMMMMMm-.hMMy/....+M:`/yNm+mMMMMMMMh    Packages: 2692 (dpkg), 108 (flatpak) 
/MMMMNmMN-:NMy`-yNMMMMMmNyyMN:`dMMMMMMM/   Shell: bash 5.1.16 
hMMMMm -odMMh`sMMMMMMMMMMs sMN..MMMMMMMh   Resolution: 1360x768, 1360x768, 1024 
NMMMMm    `/yNMMMMMMMMMMMM: MM+ mMMMMMMN   DE: MATE 1.26.0 
NMMMMm    `/yNMMMMMMMMMMMM: MM+ mMMMMMMN   WM: Metacity (Marco) 
hMMMMm -odMMh sMMMMMMMMMMs oMN..MMMMMMMh   Theme: Yaru-bark-dark [GTK2/3] 
/MMMMNNMN-:NMy`-yNMMMMMNNsyMN:`dMMMMMMM/   Icons: Yaru-bark-dark [GTK2/3] 
 hMMMMMMMm-.hMMy/....+M:.+hNd+mMMMMMMMh    Terminal: mate-terminal 
 `dMMMMMMMMy-.odNMMMMMM:    :smMMMMMMd`    Terminal Font: Ubuntu Mono 13 
   yMMMMMMMMMms/..-::oM:    .+dMMMMMy      CPU: AMD A10-5800K APU (4) @ 3.800GH 
    :mMMMMMMMMMMMMNNNNM: :smMMMMMMm:       GPU: NVIDIA Quadro K1200 
      /dMMMMMMMMMMMMMMMdNMMMMMMMd/         Memory: 4520MiB / 15938MiB 
        .odMMMMMMMMMMMMMMMMMMdo.
           `:+shmNNMMNNmhs+:`                                      
                                                                   


gnoll@ubuntubox:~$ `

I will say, I saw a comment on I THINK it was the steam discussion forums that recommended disabling the steam overlay to fix a problem, and that might've been what prevented it from doing that again when I connected to my friend a third time, but I'm not certain either way.

**Comment by gnollhide** [comment]
I couldn't get the neofetch code to play nice so I just got the system info from steam instead.

**Comment by DASPRiD** [comment]
I'm still unsure if this is actually a software problem. I did just try to install a copy of Windows on an old SSD, and I did run into blue screens there again and again at several steps during the installation process (something something clock watchdog, which on a clean install kinda points to a hardware problem).

**Comment by DASPRiD** [comment]
So, I just excluded the graphics card being the issue as well, as I tested with only the internal graphics with the GPU disconnected, and I still managed to run into a freeze.

**Comment by DASPRiD** [comment]
@gnollhide FYI, I did give it to a repairshop for diagnostics, as this really seems to be a hardware issue. I used `stress-ng` to trigger the issue consistently:

```bash
stress-ng --cpu 24 --io 2 --vm 8 --vm-bytes 5G --timeout 60s --metrics-brief
```

**Comment by gnollhide** [comment]
Huh, weird. Might be my graphics card trying to give up the ghost. I haven't had the issue since disabling steam overlay, so I dunno what it could be. I'm not an expert, but maybe something in the steam overlay is just upsetting my hardware to the point where it trips a reboot? Thank you for the information!

**Comment by DASPRiD** [comment]
But then the question arises, why it happens without even having started steam. I doubt that the steam client could be responsible for damaging hardware. On Jun 17, 2023 23:36, gnollhide ***@***.***> wrote:
I'm not certain, I've had this happen literally twice, back-to-back, when attempting to join a game in Deep Rock Galactic. Both times happened after the new client was pushed out, and does NOT seem to be happening for any linux-native games. Exact same situation, the entire desktop freezes for 5-10 seconds (with associated audio being a short, broken loop of the last thing playing), then a hard crash to BIOS and restart.
